go.mod

```diff
@@ -13,5 +13,6 @@ require (
 	github.com/nspcc-dev/neofs-api-go/v2 v2.12.0
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/zap v1.18.1
+	golang.org/x/tools v0.1.9 // indirect
```

Comment:
hmm. Neo-go might have some issues with it as they had [before](https://github.com/nspcc-dev/neo-go/pull/2276#discussion_r755362076).

Reply:
neo-go uses v0.1.8 now: https://github.com/nspcc-dev/neo-go/blob/036111d95cb25f320ac54dfeda370edac4fef05b/go.mod#L30